config/database.yml.example

```diff
@@ -4,7 +4,7 @@ default: &default
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
   username: postgres
-  password: password
```

Comment:
This shouldn't have been committed. You are supposed to only change database.yml